src/hostile/util.py

```diff
@@ -33,11 +33,11 @@ def choose_default_thread_count(cpu_count: int) -> int:
 CACHE_DIR = (
     Path(os.environ.get("HOSTILE_CACHE_DIR", ""))
     if os.environ.get("HOSTILE_CACHE_DIR")
-    else Path(user_data_dir("hostile", "Bede Constantinides"))
+    else Path(user_data_dir("hostile-eit", "Bede Constantinides"))
 )
 CPU_COUNT = multiprocessing.cpu_count()
 THREADS = choose_default_thread_count(CPU_COUNT)
-BUCKET_URL = "https://objectstorage.uk-london-1.oraclecloud.com/n/lrbvkel2wjot/b/human-genome-bucket/o"
+BUCKET_URL = "https://objectstorage.uk-london-1.oraclecloud.com/n/lr3yhdniv6gu/b/human-genome-indices/o"
```

Comment:
Someone please say if this name for the bucket is correct, chosen as the README mentions the index files in the bucket, can trivially be changed if more appropriate names.

Reply:
checked with the bucket on gpas-ltd, seems to be correct